test/integration/targets/collections_action_groups/test_collections/ansible_collections/collection_namespace/collection_name/plugins/modules/example_module.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
```

Comment:
probably better to use an echo module that returns everything it was given- also need an echo action

Reply:
Fixed. I added two action plugins, one with the same name as the module and one with a different name.